policy/modules/kernel/corenetwork.if.in

```diff
@@ -2465,6 +2451,7 @@ interface(`corenet_tcp_recvfrom_netlabel',`
 ## </param>
 #
 interface(`corenet_tcp_recvfrom_unlabeled',`
+	refpolicywarn(`$0($*) has been deprecated, please remove.')
```

Comment:
Here and below, the interfaces should not be deprecated. They are still valid, just unused.